src/sql/base/browser/ui/table/plugins/headerFilter.plugin.ts

```diff
@@ -34,6 +34,7 @@ export interface ITableFilterOptions {
 	 * The message to be displayed when the filter is disabled and the user tries to open the filter menu.
 	 */
 	disabledFilterMessage?: string;
+	autoSizePluginOn?: boolean
```

Comment:
We shouldn't tie this specifically to the autosizePlugin. Let's find a more generic name for this - maybe something like `shouldSetColumns`

Reply:
yes, we should not introduce coupling between different plugins.